packages/interface/src/components/plugin-complementary-area-header/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { close } from '@wordpress/icons';
+
+const PluginComplementaryAreaHeader = ( {
+	smallScreenTitle,
+	toggleComplementaryAreaShortcut,
+	closeComplementaryArea,
```

Comment:
How about using shorter names for those props like or something similar:
```suggestion
	toggleShortcut,
	onClose,
```

The usage `<PluginComplementaryAreaHeader onClose={ onClose } />` should have less load this way.